org.eclipse.persistence.asm/pom.xml

```diff
@@ -18,23 +18,16 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>org.eclipse.ee4j</groupId>
-        <artifactId>project</artifactId>
-        <version>1.0.6</version>
-        <relativePath/>
-    </parent>
```

Comment:
this parent definition needs to stay - it defines profiles for deployment (ie `oss-release`) among few other important things

compare running `mvn help:effective-pom` without and with this parent defined (`mvn -Poss-release -Dgpg.skip=true ...` does not work; note that `-Dgpg.skip=true` is needed to bypass signing built artifacts by gpg, required unless one has gpg configured on his box)

Reply:
> compare running mvn help:effective-pom without and with this parent defined

Is this pom used to upload to maven central? I can see that it adds sonatype distribution information. I guess this is just a requirement of https://github.com/eclipse-ee4j. I'll add it back

Reply:
> Is this pom used to upload to maven central? 

yes

>  I guess this is just a requirement[..]

No. One can use whatever he wants as long as the project gets deployed through jakarta.oss.sonatype.org repo. This common parent just makes that easier and brings in common setup and same user/dev experience to projects using it when working with other projects; ie I'm not aware of a ee4j project which does not use it